src/test/regress/expected/undistribute_table.out

```diff
@@ -0,0 +1,254 @@
+SET citus.shard_count TO 4;
+SET citus.shard_replication_factor TO 1;
+CREATE TABLE dist_table (id INT, a INT, b TEXT);
```

Comment:
can you create a custom schema, set the search_path, and drop it at the end? We do this is in many test files to clean up.

Reply:
See this PR for how: https://github.com/citusdata/citus/pull/3972